README.md

```diff
@@ -42,7 +42,10 @@ var foods = [ ["apple","banana","strawberry"], ["pizza","fries","hamburger"] ];
 Create a For Loop that outputs the following string for each piece of fruit in the console. "I want to eat a [fruit]"
 
 ```js
-// write code here
+//I know iteration is needed here, but unsure how to move forward...
+for (i = 0; i > 6; i++ ){
```

Comment:
Why 6?  How many fruits are there?  Is there a way this can adjust itself as items are added to the Array?